pkgs/applications/misc/solaar-unstable/default.nix

```diff
@@ -0,0 +1,42 @@
+{fetchFromGitHub, stdenv, gtk3, python34Packages, gobjectIntrospection}:
+python34Packages.buildPythonApplication rec {
+  name = "solaar-unstable-${version}";
+  version = "2018-02-02";
+  namePrefix = "";
+  src = fetchFromGitHub {
+    owner = "pwr";
+    repo = "Solaar";
+    rev = "master";
```

Comment:
This should be a tag or commit that will not change. This hash will change each time with master.